BackofficeBundle/Resources/public/coffee/orchestraLib.coffee

```diff
@@ -151,7 +151,78 @@ activateSelect2 = (element) ->
       container.parent().addClass('bg-color-red').attr('style', 'border-color:#a90329!important') if term.isNew
       term.text
   )
-
+#TOKENINPUT ENABLED
+activateToken = (element) ->
```

Comment:
all activateForm are in this file : activateSelect2, activateOrchestraNodeChoice...
Are you sure that we must split only for this file. Shouldn't it be a refacto ticket to split all activateForm in a particular folder? @nicolasThal